api/main.go

```diff
@@ -81,29 +81,41 @@ func checkConfig(cfg *configuration.APIRunner) error {
 }
 
 func (ar *Runner) registerServices(rpcServer *rpc.Server) error {
```

Comment:
If you make two separate `registerServices ` func (for admin and for public), it will be more clear to call one of them in `NewRunner` func